all_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Illegal on NixOS. Use `/usr/bin/env bash` instead. POSIX only mandates the presence of `/bin/sh`, not `/bin/bash`.

Reply:
Fixed